Aries/Aries/Source/Services/TransportService.swift

```diff
@@ -13,16 +13,22 @@ public class TransportService{
     
```

Comment:
We may not want to be requiring the HTTPTransport to have a completion when the WS doesn't, the end user shouldn't be able to tell if we are using HTTP or not